package.json

```diff
@@ -3,20 +3,12 @@
   "version": "0.4.2",
```

Comment:
Is it okay to leave this like this? It's a nice version number, but we are making a breaking change.

Reply:
We should update it in a followup commit, but it's slightly easier to do that separately for npm-related reasons. (At least when you have push-to-main privileges.)